CLAUDE.md

```
@@ -1,25 +1,27 @@
 # Rustrak
 
 Self-hosted error tracking, compatible with Sentry SDKs. A Rust API server with
-a small memory footprint, and an optional Next.js dashboard that can be deployed
-separately or not at all.
+a small memory footprint, and an optional dashboard the same process serves as
+static files -- or does not serve at all.
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)